README.adoc

```diff
@@ -11,9 +11,9 @@ image:https://img.shields.io/badge/License-GPLv3-blue.svg[License: GPL v3,link=h
 
 image:https://github.com/darwinia-network/rfcs/raw/master/logo/darwinia.png[image]
 
-Darwinia is a bridge blockchain connecting different chains using darwinia relay and can be connected to Polkadot network as a  parachain.
+Darwinia is a bridge blockchain connecting different chains using darwinia relay and can be connected to Polkadot network as a parachain.
 
-As an open cross-chain bridge protocol based on Substrate, Darwinia focuses on the construction of future Internet of Tokens, including decentralized tokens swap, exchange and market.
+As an open cross-chain bridge protocol based on Substrate, Darwinia focuses on the construction of future Internet of Tokens, including decentralized tokens swap, exchange, and market.
```

Comment:
`, and`? Is this OK?

Reply:
Hi @AurevoirXavier ,

Oxford commas are pretty common in American English. However, it is now not so strictly required today. I would add these commas is basically because we already use them in the current documentation (e.g., [Road Map in README](https://github.com/darwinia-network/darwinia/blob/master/README.adoc#3-road-map)). It would be nice if we can stay in one particular writing style or standard.

See also: https://en.wikipedia.org/wiki/Serial_comma

Thank you!